packages/extension-driver-redshift/project.json

```diff
@@ -0,0 +1,32 @@
+{
+  "root": "packages/extension-driver-redshift",
+  "sourceRoot": "packages/extension-driver-redshift/src",
+  "targets": {
+    "build": {
+      "executor": "@nrwl/js:tsc",
+      "outputs": ["{options.outputPath}"],
+      "options": {
+        "outputPath": "dist/packages/extension-driver-redshift",
+        "main": "packages/extension-driver-redshift/src/index.ts",
+        "tsConfig": "packages/extension-driver-redshift/tsconfig.lib.json",
+        "assets": ["packages/extension-driver-redshift/*.md"]
+      }
+    },
+    "lint": {
+      "executor": "@nrwl/linter:eslint",
+      "outputs": ["{options.outputFile}"],
+      "options": {
+        "lintFilePatterns": ["packages/extension-driver-redshift/**/*.ts"]
+      }
+    },
+    "test": {
```

Comment:
Seems missing the `publish` command. Btw, after putting the `@aws-sdk/client-redshift-data` package in the `extension-driver-redshift` vulcan package, still need the `install-dependencies` command.

Refer: https://github.com/cyyeh/vulcan-sql/blob/feature/add-redshift/packages/extension-driver-clickhouse/project.json